app.py

```diff
@@ -68,8 +76,8 @@ def getTagsFromBody():
 
 
 if __name__ == "__main__":
-    a = os.popen('hostname').read()
-    if 'Austin' not in a:
+    hostname = os.popen('hostname').read()
+    if 'Austin' not in hostname:
         app.run(host="0.0.0.0", port=5000, debug=False)
     else:
         app.run(host="0.0.0.0", port=5005, debug=False)
```

Comment:
whats up with needing to run on 5005 for ur local host?

Reply:
For my machine w docker, I had to setup port forwarding so that I could hit the docker endpoint. As a result my machine is setup to forward requests on port 5000 so I just made it locally 5005